internal/collector/collector.go

```diff
@@ -238,9 +235,18 @@ func newLogMetricsExporterContainer() *v1.Container {
 	return &exporter
 }
 
-// addSecretVolumes adds secret volumes to the pod spec for the unique set of pipeline secrets and returns the list of
+// AddSecretVolumeMounts to the collector container
+func AddSecretVolumeMounts(collector *v1.Container, secretNames []string) {
+	// List of _unique_ output secret names, several outputs may use the same secret.
+	for _, name := range secretNames {
+		path := fmt.Sprintf("/var/run/ocp-collector/secrets/%s", name)
```

Comment:
nit: We have const: https://github.com/openshift/cluster-logging-operator/blob/master/internal/constants/constants.go#L101